### PR TITLE
Add profile and a job to test Fedora Atomic in Prow

### PR DESCRIPTION
**Pull request by vrutkovs**
This depends on openshift-ansible PRs:
* [x] https://github.com/openshift/openshift-ansible/pull/10065
* [x] https://github.com/openshift/openshift-ansible/pull/10070
* [x] https://github.com/openshift/openshift-ansible/pull/10108
* [x] ~~https://github.com/openshift/openshift-ansible/pull/10126~~

Current status:
* [x] CI has `ssh-privatekey` in aws profile, but its not libra or anything uploaded to AWS account
* [x] Tests start, but too many fail. Probably VM is too slow or misconfigured

Added OWNERS file to make sure openshift-ansible can update this profile

### CODE REVIEW

**Comment by michaelgugino** [comment]
@stevekuznetsov 
@smarterclayton
@bbguimaraes
@droslean
@petr-muller 
Need approver please.

File: cluster/test-deploy/openshift-ansible-aws-atomic/vars-origin.yaml [hunk 1]
```diff
@@ -0,0 +1,119 @@
+ansible_become: true
```
Comment (smarterclayton) [inline cluster/test-deploy/openshift-ansible-aws-atomic/vars-origin.yaml]: The name of this profile should be aws-atomic, not openshift-ansible-aws-atomic

Comment (vrutkovs) [reply cluster/test-deploy/openshift-ansible-aws-atomic/vars-origin.yaml]: Renamed. The other profile has been renamed in https://github.com/openshift/release/pull/1582

File: cluster/ci/config/prow/plugins.yaml [hunk 2]
```diff
@@ -300,6 +300,8 @@ config_updater:
       name: cluster-profile-gcp-ha-static
     cluster/test-deploy/openshift-ansible-aws-rhel/*.yaml:
       name: cluster-profile-oa-aws-rhel
+    cluster/test-deploy/aws-atomic/*.yaml:
```
Comment (smarterclayton) [inline cluster/ci/config/prow/plugins.yaml]: These two don't fit the pattern.  Directory name == variable name.  Please follow the existing pattern for configmap names.

Comment (vrutkovs) [reply cluster/ci/config/prow/plugins.yaml]: Updated

File: ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml [hunk 3]
```diff
@@ -136,6 +136,59 @@ presubmits:
...
+    - release-3.10
+    - dockerless-build-temporary
+    context: ci/prow/atomic
+    decorate: true
+    name: pull-ci-openshift-openshift-ansible-release-3.10-e2e-atomic
+    rerun_command: /test atomic
+    skip_cloning: true
+    spec:
+      containers:
+      - command:
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml]: Please have `ci-operator` as your only `command` and the rest in `args`

File: ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml [hunk 4]
```diff
@@ -136,6 +136,59 @@ presubmits:
...
     trigger: ((?m)^/test( all| gcp),?(\s+|$))
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - release-3.10
+    - dockerless-build-temporary
+    context: ci/prow/atomic
+    decorate: true
+    name: pull-ci-openshift-openshift-ansible-release-3.10-e2e-atomic
+    rerun_command: /test atomic
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml]: This should be `e2e-atomic`

File: ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml [hunk 5]
```diff
@@ -136,6 +136,59 @@ presubmits:
           - configMap:
               name: cluster-profile-gcp
     trigger: ((?m)^/test( all| gcp),?(\s+|$))
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - release-3.10
+    - dockerless-build-temporary
+    context: ci/prow/atomic
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml]: This should be `ci/prow/e2e-atomic`

File: ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml (hunk 4 above)
Comment (vrutkovs) [reply ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml]: Fixed

File: ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml (hunk 5 above)
Comment (vrutkovs) [reply ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml]: Fixed

File: ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml (hunk 3 above)
Comment (vrutkovs) [reply ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml]: Done

**Pull request merged**